dom.bs

```diff
@@ -6958,8 +6960,20 @@ a boolean <var>delegatesFocus</var>, and a boolean <var>slotAssignment</var>:
   <p>If <var>element</var> is a <a for=Element>shadow host</a>, then:
 
   <ol>
-   <li><p>If <var>element</var>'s <a for=Element>shadow root</a>'s <a for=ShadowRoot>declarative</a>
-   is false, then <a>throw</a> an "{{NotSupportedError!!exception}}" {{DOMException}}.
+   <li><p>Let <var>currentShadowRoot</var> be <var>element</var>'s
+   <a for=Element>shadow root</a>.
+
+   <li>
+    <p>If one of the following is true:
+
+    <ul>
+     <li><p><var>currentShadowRoot</var>'s <a for=ShadowRoot>declarative</a> is false,
+
+     <li><p><var>currentShadowRoot</var>'s <a for=ShadowRoot>mode</a> does not match
+     <var>mode</var>, or
```

Comment:
Is there something missing here, or should the trailing _or_ be removed?

Reply:
It should move up one bullet point I think.

Reply:
Fixed. I removed the `, or` because I couldn't find other examples. For example, https://dom.spec.whatwg.org/#retarget just has the bullet list of options. LMK if that's the wrong conclusion to reach.